psi/__init__.py

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
@xqliu, putting some "global" constants in the package init so we can import them later on. This way there's a single definition.

Ideally, the `os.path.join` calls shouldn't happen deep in the code, but it would require another PR to factor all of that out